content/en/docs/tutorials/backup.md

```diff
@@ -49,5 +57,109 @@ created above after installing cert-manager.
 $ kubectl apply -f cert-manager-backup.yaml
 ```
 
-If you have migrated from an old cluster, you will need to make sure to run a
-similar `kubectl apply` command to restore your Secret resources too.
+## Full cluster backup and restore
+
+This section refers to backing up and restoring 'all' Kubernetes resources in a
+cluster — including some `cert-manager` ones — for scenarios such as disaster
+recovery, cluster migration etc.
+
+*Note*: We have tested this process on simple Kubernetes test clusters with a limited set of Kubernetes releases. To avoid data loss, please test both the backup and the restore strategy on your own cluster before depending upon it in production. If you encounter any errors, please open a GitHub issue or a PR to document variations on this process for different Kubernetes environments. 
+
+### Avoiding unnecessary certificate reissuance
+
+#### Order of restore
+
+If `cert-manager` does not find a Kubernetes `Secret` with an X.509 certificate
+for a `Certificate`, reissuance will be triggered. To avoid unnecessary
+reissuance after a restore, ensure that `Secret`s are restored before
+`Certificate`s. Similarly, `Secret`s should be restored before `Ingress`es if you
+are using [`ingress-shim`](../../usage/ingress/).
+
+#### Excluding some cert-manager resources from backup
+
+`cert-manager` has a number of resources- `Order`s, `Challenge`s and
+`CertificateRequest`s -that are designed to represent a point-in-time operation
+such as a request for a certificate. As such, their status often depends on other
+ephemeral resources (i.e a temporary Secret holding a private key), so
+`cert-manager` cannot always correctly re-create the status of these resources
+by just looking at the cluster state at some later time.
```

Comment:
```suggestion
`cert-manager` has a number of resources — `Order`s, `Challenge`s and
`CertificateRequest`s  — that are designed to represent a point-in-time operation
such as a request for a certificate. As such, their status often depends on other
ephemeral resources (i.e a temporary Secret holding a private key), so
`cert-manager` cannot always correctly re-create the status of these resources
by just looking at the cluster state at some later time.
```

Reply:
Thanks @wallrj ! I have eliminated those dashes by re-writing this using shorter separate sentences here and in a few more places 😄